src/components/utils/jwt.ts

```diff
@@ -31,6 +31,8 @@ export const getEnterpriseFeaturesFromJWT = (
 ): EnterpriseFeatures => {
   try {
     const jwt = parseJwt(token)
+    // @ts-ignore
+    jwt.enterprise_features.logoCobrand = true
```

Comment:
You can just return `{...jwt.enterprise_features, logoCobrand: true}` in the line below and remove this line

Reply:
So we want the `logoCobrand` to be true for all `enterprise_features`?

Reply:
Lol my apologies, I just did this for testing, thanks for catching! Updated.